sky/templates/kubernetes-port-forward-proxy-command.yaml.j2

```diff
@@ -4,11 +4,20 @@ set -uo pipefail
 
 # Ignore this function and `e home` if you use something like `kubectx` to manage your kubectl contexts and environments
 e(){
-    export KUBECONFIG="/home/gcpuser/.kube/config"
+    export KUBECONFIG="/home/`whoami`/.kube/config"
 }
 
 e home
 
+POD_NAME=$(kubectl get pods --selector=skypilot-cluster={{ cluster_name }},ray-node-type=head -o jsonpath='{.items[0].metadata.name}' 2> /dev/null)
+if ! [ -z "$POD_NAME" ]; then
+    kubectl exec $POD_NAME -- bash -c "printenv > /home/sky/.k8s_env" 2> /dev/null
```

Comment:
This approach with the symlink we create between `.k8s_env` and `/etc/environment` currently overwrites what's stored in `/etc/environment` each time the user `ssh` in to the k8s instance. The user may have set a a system-wide env var for their use in `/etc/environment`, and this value will get wiped as running `ssh <cluster-name>` with the current implementation. Is there a way to reserve the env vars users set in `/etc/environment`?

Reply:
Can we  use `/etc/profile.d/skypilot_k8s_envvars.sh` to store these environment variables? [Reference](https://askubuntu.com/a/866169)

Reply:
We should also check if [`_update_envs_for_k8s`](https://github.com/skypilot-org/skypilot/blob/c083f20886fa545366eddafb32d028e116da7c18/sky/backends/cloud_vm_ray_backend.py#L3001-L3022) is required anymore after this (since we'd populate envvars on every ssh connection)